code/mathematical_algorithms/src/tribonacci_numbers/tribonacci_numbers.js

```diff
@@ -0,0 +1,14 @@
+function tribonacci(n) {
+  if (n == 0) {
+    return 0;
+  }
+  else if (n == 1) {
+    return 0;
+  }
+  else if (n == 2) {
+    return 1;
+  }
+  return tribonacci(n-3) + tribonacci(n-2) + tribonacci(n-1);
```

Comment:
One more minor change before this is good to merge: Add spaces around all operators please.

Reply:
@AdiChat @arnavb 

I fixed this. It should be good to merge now